src/main/java/com/jcabi/github/RtIssue.java

```
@@ -150,17 +156,42 @@ public void react(final Reaction reaction) {
 
     @Override
     public void lock(final String reason) {
-        throw new UnsupportedOperationException("lock not implemented");
+        final JsonStructure json = Json.createObjectBuilder()
+            .add("lock_reason", reason)
+            .build();
+        try{
+            this.request.method(Request.PUT).uri().path("/lock").back()
+                .body().set(json).back()
+                .fetch()
+                .as(RestResponse.class)
+                .assertStatus(HttpURLConnection.HTTP_NO_CONTENT);
+        } catch (IOException error){}
```

@paulodamaso why swallow the exception?